lib/ex_unit/test/ex_unit/assertions_test.exs

```diff
@@ -253,6 +253,16 @@ defmodule ExUnit.AssertionsTest do
       "(undefined function: Not.Defined.function/3 (module Not.Defined is not available))" = error.message
   end
 
+  test "assert raise with some other error includes stacktrace from original error" do
+    "This should never be tested" = assert_raise ArgumentError, fn ->
+      Not.Defined.function(1, 2, 3)
+    end
+  rescue
+    error in [ExUnit.AssertionError] ->
```

Comment:
`error` variable is unused, which generates a compile time warning. Can remove the variable:
```elixir
rescue
  ExUnit.AssertionError ->
```